metrics2/prometheus2/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
```

Comment:
Why is there a Prometheus 2 - this does not depend on MP metrics at all, only on prometheus itself.
This module should not have been duplicated.

Reply:
See the earlier comment about ease of removing the older `metrics` and `microprofile/metrics` modules later.